File: FrameworkLib_Common/Model/Crow.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace FrameworkLib_Common.Model
{
    //[Table("Crow")]
    public class Crow : Animal
    {
        public string Color { get; set; } = null!;

        [NotMapped]
        public int Size { get; set; }

        public override string ToString()
        {
            return @$"{base.ToString()} Crow : Color = {this.Color} Size = {this.Size} (cause [NotMapped])";
        }
    }
}

File: VBLib_Common/Model/Crow.vb
Imports System.ComponentModel.DataAnnotations.Schema

Namespace Model
    Public Class Crow
        Inherits Animal

        Public Property Color As String
        <NotMapped>
        Public Property Size As Integer

        Public Overrides Function ToString() As String
            Return _
                $"{MyBase.ToString()} Crow : Color = {Me.Color} Size = {Me.Size} (cause [NotMapped])"
        End Function
    End Class
End Namespace
